js/web/lib/onnxjs/session.ts

```diff
@@ -16,6 +16,7 @@ export declare namespace Session {
   export interface Config {
     backendHint?: string;
     profiler?: Profiler.Config;
+    logger?: Logger.Config;
```

Comment:
it is not a good design to add logger option in `Session.Config`. The reason is, we don't really have a per-session logger in ONNX.js. The logger in ONNX.js is a global singleton, so even this config is added, it does not work for multiple sessions. The actual logger only applies the latest change.

the interface `Env` in onnxruntime-common is the best place to setup logging options. we should add it here : https://github.com/microsoft/onnxruntime/blob/b4e8e9b004cb3a5b5f4b9cd59212fe7704e70538/js/common/lib/env.ts#L41

Reply:
this has been taken out, will have another pr with logger related changes.